core/crypto/bls/bls_types.hpp

```diff
@@ -6,17 +6,15 @@
 #ifndef CRYPTO_BLS_PROVIDER_TYPES_HPP
 #define CRYPTO_BLS_PROVIDER_TYPES_HPP
 
-#include <filecoin-ffi/filecoin.h>
 #include <array>
 
 #include "common/outcome.hpp"
 
 namespace fc::crypto::bls {
-
-  using PrivateKey = std::array<uint8_t, PRIVATE_KEY_BYTES>;
-  using PublicKey = std::array<uint8_t, PUBLIC_KEY_BYTES>;
-  using Signature = std::array<uint8_t, SIGNATURE_BYTES>;
-  using Digest = std::array<uint8_t, DIGEST_BYTES>;
```

Comment:
If so bls_types should be removed and filecoinf-ffi dependency should be moved to bls_provider

Reply:
in CMakeLists